build-tools/packages/build-tools/package.json

```diff
@@ -45,6 +45,7 @@
 		"cosmiconfig": "^8.3.6",
 		"date-fns": "^2.30.0",
 		"debug": "^4.3.4",
+		"depcheck": "^1.4.7",
```

Comment:
Does the build-tools package need the dependency? I get it in build-cli but not sure what it's doing here.

Reply:
build-tools doesn't. Just the build-cli's dependencies should suffice.